apps/jsii-docgen/test/__fixtures__/libraries/construct-library/package.json

```diff
@@ -24,28 +24,26 @@
     "email": "dummy@example.com",
     "organization": false
   },
-  "dependencies": {
-    "@aws-cdk/aws-events": "^1.204.0",
-    "@aws-cdk/aws-iam": "^1.204.0",
-    "@aws-cdk/aws-kms": "^1.204.0",
-    "@aws-cdk/aws-s3": "^1.204.0",
-    "@aws-cdk/cloud-assembly-schema": "^1.204.0",
-    "@aws-cdk/core": "^1.204.0",
-    "constructs": "^3.4.344"
-  },
   "peerDependencies": {
     "@aws-cdk/aws-events": "^1.204.0",
     "@aws-cdk/aws-iam": "^1.204.0",
     "@aws-cdk/aws-kms": "^1.204.0",
     "@aws-cdk/aws-s3": "^1.204.0",
     "@aws-cdk/cloud-assembly-schema": "^1.204.0",
     "@aws-cdk/core": "^1.204.0",
-    "constructs": "^3.3.69"
+    "constructs": "^3.4.344"
   },
   "devDependencies": {
+    "@aws-cdk/aws-events": "1.204.0",
+    "@aws-cdk/aws-iam": "1.204.0",
+    "@aws-cdk/aws-kms": "1.204.0",
+    "@aws-cdk/aws-s3": "1.204.0",
+    "@aws-cdk/cloud-assembly-schema": "1.204.0",
+    "@aws-cdk/core": "1.204.0",
+    "constructs": "3.4.344",
     "@types/node": "^18.17.13",
     "@types/prettier": "2.6.0",
-    "jsii": "~5.0.21",
+    "jsii": "~5.3.11",
     "typescript": "^4.9.5"
```

Comment:
Moved stuff around here to avoid the annoying jsii warnings about needing devDeps